sources/Models/Vector2.hpp

```diff
@@ -129,6 +129,11 @@ namespace BBM
 		{
 			double mag = this->magnitude();
```

Comment:
if sqrt not needed please remove it

Reply:
What are you referring to? There is no sqrt here.

Reply:
inside the function call